react/features/video-quality/components/VideoQualitySlider.web.js

```diff
@@ -56,6 +56,12 @@ type Props = {
      */
     _receiverVideoQuality: Number,
```

Comment:
doesn't this need to go?

Reply:
Thanks for the review Saul, I missed this one, remove it now.